### PR TITLE
feat: #id 22723 wrap appCatalog

### PR DESCRIPTION
**Pull request by amaroks**
https://chartiq.kanbanize.com/ctrl_board/18/cards/22723/details/

Testing:
1. Checkout seed@`22723-wrap-appcatalog`
1. In toolbar's devtools run the following command to spawn app catalog
```javascript
FSBL.Clients.LauncherClient.spawn("App Catalog")
```

### CODE REVIEW

File: src-built-in/components/appCatalog/src/app.jsx [hunk 1]
```diff
@@ -9,47 +9,8 @@ import ReactDOM from "react-dom";
...
-				label: "Apps",
-				content: <AppContent installed={true} />
-			}]} />
-		);
-	}
-}
+import "../../complexMenu/menu.css";
+import { AppCatalog } from "@chartiq/finsemble-ui/src/components/appCatalog/appCatalog";
 
 if (window.FSBL && FSBL.addEventListener) { FSBL.addEventListener("onReady", FSBLReady); } else { window.addEventListener("FSBLReady", FSBLReady) }
```
Comment (zona-ciq) [inline src-built-in/components/appCatalog/src/app.jsx]: Can we hide the FSBLReady code behind finsemble-ui? This code has no use to the client developers and they probably don't have to see it

Comment (carterbs) [reply src-built-in/components/appCatalog/src/app.jsx]: Agreed! Use the onReady hook. Make one if there isn't one. Then you can just render the component.

Comment (amaroks) [reply src-built-in/components/appCatalog/src/app.jsx]: You can not use hooks outside a component function. Where do you want to use the hook?

It makes no sense to move the following into finsemble-ui
```javascript
	ReactDOM.render(
		<AppCatalog />
		, document.getElementById("bodyHere"));
```
Create another component function? Please make it clear what changes/where you want them to be.

Comment (zona-ciq) [reply src-built-in/components/appCatalog/src/app.jsx]: I created a component to wrap around the component we're wrapping and only renders it when FSBL is ready: https://github.com/ChartIQ/finsemble-ui/pull/11/files#diff-5d2eca232b4030fea5fff9ca17c73701R126
Eventually we'll refactor this component to a helper function so everyone else can use it

**Comment by amaroks** [comment]
All good here? @zona-ciq I didn't delete anything.

**Pull request closed without merging**